crates/cli/tests/run.rs

```diff
@@ -27,9 +27,18 @@ where
 fn test_proc_exit() {
     let mut cmd = get_cmd();
     let assert = cmd.arg(get_bin_path("proc_exit")).assert();
+    assert!(assert.get_output().stdout.is_empty());
     assert.failure().code(1);
 }
 
+#[test]
+fn test_verbose() {
+    let mut cmd = get_cmd();
+    let assert = cmd.arg(get_bin_path("proc_exit")).arg("--verbose").assert();
+    let stdout = &assert.get_output().stdout;
+    assert!(contains_slice(stdout, b"proc_exit.wat\")::()"));
```

Comment:
Purposely chose some text to check that seems likely to be stable across slight path changes or i18n, by the way.